### PR TITLE
No one can view or edit with a invite code that has been fully submitted

### PR DESCRIPTION
**Pull request by jeffmaher**
# Description 

Working on [Trello card](https://trello.com/c/MOaKInr8/498-noone-can-edit-the-report-after-it-has-been-submitted), which adds an error message when a doctor tries to login after the medical report has been fully submitted.

# Instructions

1. Fill out the applicant sections to generate an application code
1. Login to the medical pro view, and complete and submit a full report
1. Change the URL to `/en/dashboard` and you should get directed back to the medical professional login page (if it goes to the dashboard, this failed)
1. Try to login with the invite code again, and you should get an error

# Notes

- Updated the display of error messages across the app to match the design on this page's Figma. @cds-snc/esdc-cppd-design : Let me know if that's the wrong thing to do. You can see the look also by trying to submit any form in the app and seeing the ❗️ icon.

### CODE REVIEW

File: config/locales/en.json [hunk 1]
```diff
@@ -619,5 +619,14 @@
...
+	"errors.phone_required": "Phone is required",
+	"declaration.review_and_confirm": "Review and confirm the contents of this medical report before submitting to Service Canada.",
+	"edit": "Edit",
+	"declaration.condition_likely_to": "%s's condition is likely to",
+	"declaration.you_are_a": "You are a",
+	"phone_number": "Phone number",
+	"declaration.submit_to_service_canada": "Submit report to Service Canada",
+	"declaration.i_confirm_that": "I confirm that, to the best of my knowledge all of the information I have provided in this report is accurate and complete",
+	"declaration.title": "Declaration and review",
+	"errors.application_submitted_already": "A medical report has already been submitted using the unique code you entered and connect be accessed. Call Service Canada at 1-800-277-9914 if you have any questions."
```
Comment (keithiopia) [inline config/locales/en.json]:
```suggestion
	"errors.application_submitted_already": "A medical report has already been submitted using the unique code you entered and cannot be accessed. Call Service Canada at 1-800-277-9914 if you have any questions."
```

**Review by keithiopia — CHANGES_REQUESTED** [review]
Small typo.

Also, I'm shown the error message the first time I enter the invitation code and date of birth.

**Review by kylejones-ESDC — COMMENTED** [review]
I am receiving the error code in question "A medical report has already been submitted using the unique code you entered and connect be accessed. Call Service Canada at 1-800-277-9914 if you have any questions." when attempting to begin the medical report, prior to having ever completed a medical report using that code.

**Comment by jeffmaher** [comment]
Thanks for the feedback! Will investigate.

**Comment by jeffmaher** [comment]
I think I got the bug fixed! Ready for re-review from @keithiopia and @kylejones-ESDC .

**Review by kylejones-ESDC — DISMISSED** [review]
Some small bugs noted, Trello cards created.

The stated functionality works as indicated.

**Review by keithiopia — DISMISSED** [review]
Looks about right to me!

**Review by CalvinRodo — DISMISSED** [review]
LGTM!

**Review by dsamojlenko — APPROVED** [review]

**Pull request merged**